user_guide/content/modules/red_hat_openshift_io_overview.adoc

```diff
@@ -5,15 +5,37 @@
 
 == Features
 
-{osio} offers features for seamless management of end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
+{osio} offers features to seamlessly manage end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
+
+*Project Management:*
 
 * Project planning and issue tracking using your choice of Agile methodology.
-* Creating development environments and testing requirements.
-* Coding using an integrated development environment based on Eclipse Che.
-* Building, testing, deploying, and monitoring codebases using pipelines.
+* Automating releases of development projects.
+
+*Project Development:*
+
+* Creating development environments and testing based on requirements.
+* Coding using an integrated development environment based on Eclipse Che, which provides a workspace for developing your application.
+* Development in this context-aware environment provides the appropriate context for the developer and project, such as which branch, repository, language, and files are referenced.
 * Interacting with integrated GitHub repositories.
 * Compiling and reviewing code in Pull Requests as required.
-* Automating releases of development projects.
```

Comment:
I'm not sure what we mean by "development project." Is this meant to refer to projects that are in development, but not yet released? (Maybe we should just refer to a "project."

Reply:
That was indeed the intention. Changed for clarity.